modules/manage/pages/manage-security/configure-ldap.adoc

```diff
@@ -266,14 +300,17 @@ When the *LDAP Query* radio-button is selected, the *Query for Groups Using* pan
 image::manage-security/ldapConfigurationLDAPquery.png[,400,align=left]
 
 Selection of *LDAP Query* instructs Couchbase Server to perform an LDAP search, in order to retrieve a list of the user's groups.
-For explanations of *Base*, *Filter*, *Scope*, see https://ldap.com/the-ldap-search-operation/[The LDAP Search Operation^].
-When the search is conducted, `%u` is replaced with the specified username; and
-`%D` is replaced with the user's DN.
+*Base* is the _search base_ https://ldap.com/the-ldap-search-operation/[DN].
+*Filter* is the https://ldap.com/the-ldap-search-operation/[search filter].
```

Comment:
Probably this link is better for 'filter': https://ldap.com/ldap-filters/

Reply:
Have changed the link.